reconcile/queries.py

```diff
@@ -5,13 +5,32 @@
 import shlex
 
 from textwrap import indent
-from typing import Any, Optional
+from typing import Any, Mapping, Optional
 
 from jinja2 import Template
 
 from reconcile.utils import gql
 
 
+SECRET_READER_SETTINGS = """
+{
+  settings: app_interface_settings_v1 {
+    vault
+  }
+}
+"""
```

Comment:
If its not too much of a hassle, can we use our new query class generator for this? 

I can also have a look at it after this is merged.